az-core/src/main/java/azkaban/utils/MemConfValue.java

```diff
@@ -0,0 +1,45 @@
+package azkaban.utils;
+
+import azkaban.Constants;
+import com.google.common.base.Preconditions;
+import org.apache.commons.lang.StringUtils;
+
+public class MemConfValue {
```

Comment:
Can this be applied to other properties? If so, can we make this universal?

Reply:
How? I guess these are the only properties that azkaban parses expecting this pattern.